src/Microsoft.Azure.SignalR.Common/DefaultCultureFeatureManager.cs

```diff
@@ -37,6 +37,12 @@ public bool TryRemoveCultureFeature(string requestId, out IRequestCultureFeature
         return false;
     }
 
+    public bool IsDefaultFeature(IRequestCultureFeature feature)
+    {
+        // this is the default feature value in blazor when no culture feature is configured by app server
+        return feature.RequestCulture.Culture == CultureInfo.DefaultThreadCurrentCulture && feature.RequestCulture.UICulture == CultureInfo.DefaultThreadCurrentUICulture && feature.Provider is AcceptLanguageHeaderRequestCultureProvider;
```

Comment:
is AcceptLanguageHeaderRequestCultureProvider does not mean it is default though